File: fromJAVAtoPYTHON/04_OOP/04.Exercise_Part_1_Add_an_attribute/Book.py
class Book:
    # constructor
    def __init__(self, name, copies) -> None:
        self.name2 = name
        self.copies2 = copies

    #toString
    def __repr__(self) -> str:
        return repr((self.name2, self.copies2))

#instances
book2 = Book('John', 200)
book3 = Book('Mike', 15)

print(book2)
print(book3)

File: fromJAVAtoPYTHON/04_OOP/04.Exercise_Part_1_Add_an_attribute/Book.java
public class Book {
    String name;
    int copies;

    Book(String name, int copies) {
        this.name = name;
        this.copies = copies;
    }

    public String toString() {
        return "(" + name + ", " + copies + ")";
    }

    public static void main(String[] args) {
        Book book1 = new Book("John", 200);
        Book book2 = new Book("Mike", 15);

        System.out.println(book1);
        System.out.println(book2);
    }
}